README.md

```diff
@@ -33,6 +35,26 @@ function YourComponent() {
     </div>
   );
 ```
+### Import Javascript Bundle
+
+This is a framework agnostic method that can be used in any JavaScript project. The `CioQuiz` function will mount a Quiz to the element for the given selector and it will handle state management, data fetching, and rendering logic for the entire quiz.
```

Comment:
I don't love

> The `CioQuiz` function will mount a Quiz to the element for the given selector and it will handle state management, data fetching, and rendering logic for the entire quiz.

but I couldn't find anything better. @jjl014 feel free to edit directly

Reply:
Maybe we could re-use the description from our docs:

> Allows for a simple interface to inject a quizzes UI into the DOM directly simply by supplying a container reference.

But reword it to fit a bit better for the repo:

> The `CioQuiz` function provides a simple interface to inject an entire Quizzes UI into the provided `selector`.

Probably no necessary to mention state management, data fetching, etc here. Maybe it could be something we can add to the library description at the top of the README instead